playwright-e2e/package.json

```diff
@@ -30,12 +30,16 @@
     "uuid": "^9.0.0"
   },
   "dependencies": {
+    "@google-cloud/local-auth": "^2.1.0",
```

Comment:
Added google auth, APIs and mailparser so that we can read email from a forwarded inbox, parse it, and grab various fields from emails to validate email sending.